pypeclub/OpenPype
# Module: Kitsu module

## Brief description
[Kitsu](https://www.cg-wire.com/fr/kitsu.html) module.

## Description
Use Kitsu as project management module.

## Documentation (add _"type: documentation"_ label)
Doc has been written, you can read it in PR files.

## Testing notes:
1. Fill `Server` url in Project Settings.
2. When launching Tray, `Kitsu Credentials` window will pop. Either way click on `Kitsu Connect` button and sign-in.
3. Run `.poetry/bin/poetry run python start.py module kitsu sync-service -l me@domain.ext -p mypass` to start sync OP with Kitsu, events listening will start right after sync is achieved.
4. Publish a new version of an asset and see the magic happening.

You can also try `.poetry/bin/poetry run python start.py module kitsu push-to-zou -l me@domain.ext -p mypass`, but it cannot handle all use cases and must be considered as a helpful facultative utility.

Listening currently supports most of _project/episode/sequence/shot/asset/task create/update/delete_ events, but not `project:delete`, `episode:delete`, `sequence:delete` because of a bug, which will be resolved by https://github.com/cgwire/zou/pull/427
